direct_webapp/urls.py

```diff
@@ -18,7 +18,10 @@
 from django.contrib import admin
 from django.urls import include, path
 
+from main import views
+
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("main.urls")),
+    path("about/", views.AboutPageView.as_view(), name="about"),
```

Comment:
This should be added to the url patterns in `main/urls.py` instead of this file.

Reply:
Done.